app/Http/Libraries/Requests/Cache/TerritoryList.php

```diff
@@ -27,11 +27,11 @@ public function generate(): array
 
                 $territory = [];
                 $territory['territory'] = $item['territory'];
-                $territory['guild'] = $guild->id;
-                $territory['guildPrefix'] = $guild->prefix;
```

Comment:
iirc, the `$guild->prefix` was calculated and populated into a guild map at some other place in Athena. That seems like it could now be removed, as well, since I guess this was the only use of this prefix.